core/src/main/java/org/bitcoinj/wallet/DeterministicKeyChain.java

```diff
@@ -182,6 +185,45 @@ private int calcDefaultLookaheadThreshold() {
     // and always 1 for other transaction types
     protected int sigsRequiredToSpend = 1;
 
+    @Override
+    public Descriptor descriptor() {
+        // TODO: Don't hardcode network parameters
+        return HdDescriptor.of(this, MainNetParams.get());
```

Comment:
I'm not sure what we should do about this. Maybe add a `NetworkParameters` parameter to `descriptor()`? Or move the implementation of `Describable` to `KeychainGroup` which does contain `NetworkParameters` information. (It would have to return a list of `Descriptors`, though...

Reply:
Generation of descriptors has moved to `keyChainGroup` and `Describable` is currently unused.